projects/angular/src/forms/styles/_checkbox.clarity.scss

```diff
@@ -106,14 +108,27 @@
       border-left: none;
       @include mixins.css-var(
         border-bottom-color,
-        $indetermCssBrdrColorVar,
-        $indetermSassBrdrColorVar,
+        clr-forms-checkbox-indeterminate-border-color,
+        forms-variables.$clr-forms-checkbox-indeterminate-border-color,
         variables.$clr-use-custom-properties
       );
       display: inline-block;
       transform: translate(0, $mark-half-size);
     }
 
+    input[type='checkbox'].clr-indeterminate:checked + .clr-control-label::after,
+    input[type='checkbox']:indeterminate:checked + .clr-control-label::after {
+      border-left: forms-variables.$clr-forms-checkbox-mark-border-size solid;
+      border-bottom: forms-variables.$clr-forms-checkbox-mark-border-size solid;
+      @include mixins.css-var(
+        border-color,
+        clr-forms-checkbox-mark-color,
+        forms-variables.$clr-forms-checkbox-mark-color,
+        variables.$clr-use-custom-properties
+      );
+      transform: translate(0, $mark-half-size) rotate(-45deg);
+    }
```

Comment:
Is that needed? Basically it doesn't matter for the checked state if it's `indeterminate` or not and it's already working without these lines.

Reply:
corrected in commit: 628b080